.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+on: [release]
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [3.8]
+    
+    steps:
+    - uses: actions/checkout@master
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+        
+    - name: Install pypa/build
+      run: python -m pip install setuptools build wheel --user
+
+    - name: Build a binary wheel
+      run: python setup.py sdist bdist_wheel
+        
+    - name: Publish distribution 📦 to Test PyPI
+      uses: pypa/gh-action-pypi-publish@master
+      with:
+        password: ${{ secrets.TEST_PYPI_API_TOKEN }}
+        repository_url: https://test.pypi.org/legacy/
+        skip_existing: true
+        
+    - name: Publish distribution 📦 to PyPI
+      if: startsWith(github.ref, 'refs/tags')
```

Comment:
Is this going to push only the latest tag?

Reply:
Yes, only the branch or tag, and I'm using startsWith to lock it into the tag, not the branches. 

`github.ref` is the... well, I'll let these folks say it better. 😃 

<html>
<body>
<!--StartFragment-->

github.ref | string | The branch or tag ref that triggered the workflow run. For branches this is the format refs/heads/<branch_name>, and for tags it is refs/tags/<tag_name>.
-- | -- | --


<!--EndFragment-->
</body>
</html>

https://docs.github.com/en/actions/learn-github-actions/contexts#github-context